src/streams.h

```diff
@@ -430,9 +430,18 @@ class AutoFile
     /** Implementation detail, only used internally. */
     std::size_t detail_fread(Span<std::byte> dst);
 
+    /** Wrapper around fseek(). Will throw if seeking is not possible. */
     void seek(int64_t offset, int origin);
+
+    /** Find position within the file. Will throw if unknown. */
     int64_t tell();
```

Comment:
Slightly unrelated, but since we're modifying its usage here: `streams_tests` doesn't seem to cover this method at all. 

(+ Commit, Truncate and many `std::ios_base::failure` throws seem also uncovered)